packages/tree-select/test/index.js

```diff
@@ -221,15 +221,24 @@ describe( 'index', () => {
 			expect( afterClearResult ).not.toBe( firstResult );
 		} );
 
-		test( 'should memoize a falsy value returned by getDependents', () => {
-			const memoizedSelector = treeSelect( () => [ null ], () => [] );
+		test( 'should memoize a nullish value returned by getDependents', () => {
+			const memoizedSelector = treeSelect( () => [ null, undefined ], () => [] );
 			const state = {};
 
 			const firstResult = memoizedSelector( state );
 			const secondResult = memoizedSelector( state );
 			expect( firstResult ).toBe( secondResult );
 		} );
 
+		test( 'throws on a non-nullish primitive value returned by getDependents', () => {
+			[ true, 1, 'a', false, '', 0 ].forEach((primitive) => {
+				const memoizedSelector = treeSelect( () => [ primitive ], () => [] );
+				const state = {};
+
+				expect(() => memoizedSelector( state )).toThrow();
+			});
```

Comment:
This test code still needs the correct `wp-prettier` formatting: more spaces inside parens and no parens around an arrow function argument.

This should do the job:
```
./node_modules/bin/prettier --write  packages/tree-select/test/index.js
```